docs/upgrade.html

```diff
@@ -28,6 +28,8 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
     <li>The Streams API removed all deprecated APIs that were deprecated in version 2.5.0 or earlier.
         For a complete list of removed APIs compare the detailed Kafka Streams upgrade notes.</li>
     <li>The deprecated <code>Metric#value()</code> method was removed (<a href="https://issues.apache.org/jira/browse/KAFKA-12573">KAFKA-12573</a>).</li>
+    <li>The deprecated <code>org.apache.kafka.clients.admin.ConfigEntry</code> constructor was removed (<a href="https://issues.apache.org/jira/browse/KAFKA-12577">KAFKA-12577</a>).
```

Comment:
Do we need the fully qualified name? We didn't use it for the `Metric` case.

Reply:
We have multiple `ConfigEntry` classes in the code base so I used the fully qualified name to be precise. `Metric` is unique however. Shall we use the fully qualified name for `Metric` as well in order to be consistent?

Reply:
Do we have multiple `ConfigEntry` that are public APIs?  One option would be to link to the file in GitHub using a commit hash from the 2.8 branch. That would be both concise and informative (the deprecation documentation would include the suggested replacement typically).

Reply:
On the other hand, it won't be practical to do the same for cases where there are a lot of deprecated usages. So, maybe we should keep it simple and people should make sure to remove deprecations before the upgrade so they get useful help.

Reply:
> Do we have multiple `ConfigEntry` that are public APIs?

Good point. There is only one in the public APIs. Let me remove the fully qualified name.